File: crates/dashboard-model/src/victron_controller/dashboard/from_0_1_0_command_set_bool_knob.rs


pub fn convert__command__set_bool_knob__from__0_1_0(from: &crate::victron_controller::dashboard::v0_1_0::command::SetBoolKnob) -> crate::victron_controller::dashboard::command::SetBoolKnob {
    crate::victron_controller::dashboard::command::SetBoolKnob {
        knob_name: from.knob_name.clone(),
        value: from.value.clone(),
    }
}

File: web/src/model/victron_controller/dashboard/v0_2_0/from_0_1_0_command_set-bool-knob.ts
// @ts-nocheck
import {SetBoolKnob as v0_1_0_SetBoolKnob} from '../v0_1_0/Command'
import {SetBoolKnob as v0_2_0_SetBoolKnob} from './Command'

export function convert__command__set_bool_knob__from__0_1_0(from: v0_1_0_SetBoolKnob): v0_2_0_SetBoolKnob {
    return new v0_2_0_SetBoolKnob (
        from.knob_name,
        from.value
    )
}